src/pipecat/services/cartesia/tts.py

```diff
@@ -101,16 +121,19 @@ class InputParams(BaseModel):
 
         Parameters:
             language: Language to use for synthesis.
-            speed: Voice speed control.
-            emotion: List of emotion controls.
+            speed: Voice speed control for non-Sonic-3 models (literal values).
+            emotion: List of emotion controls for non-Sonic-3 models.
 
                 .. deprecated:: 0.0.68
                         The `emotion` parameter is deprecated and will be removed in a future version.
+            generation_config: Generation configuration for Sonic-3 models. Includes volume,
```

Comment:
Add a newline before `generation_config` so that docstrings are correctly created.

```suggestion

            generation_config: Generation configuration for Sonic-3 models. Includes volume,
```